package.json

```diff
@@ -64,9 +50,7 @@
     "requestretry": "^4.0.2",
     "socket.io-client": "^2.2.0",
     "stats-lite": "https://registry.npmjs.org/stats-lite/-/stats-lite-2.0.3.tgz",
-    "swagger-node-express": "^2.1.3",
```

Comment:
in use

Reply:
This doesn't appear to be in use outside the `api` directory, which has its own `package.json` file:

https://github.com/firewalla/firewalla/blob/4550b0756ce72d253290d437c5f7d54a8d213005/api/package.json#L19